exectoken/src/main/java/io/redlink/utils/spring/exectoken/ExecutionToken.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 Redlink GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.redlink.utils.spring.exectoken;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Date;
+import java.util.Map;
+
+import org.bson.types.ObjectId;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
Why do we have a dependency on Jackson here?

Reply:
so that Tokens are correctly serialized if shared over JSON